README.rst

```diff
@@ -28,6 +28,14 @@ Under the hood this tool provides interfaces to access the following features of
 * Writing an I2C Configuration-EEPROM on the configuration I2C.
   This is done using an undocumented command that was reverse-engineered from Microchip's freely available EOL-Tools.
 
+Debian Package
+--------------
+
+This tool is `packaged <https://packages.debian.org/search?suite=all&searchon=names&keywords=usbsdmux>`_ in Debian
+*bookworm* and later.
```

Comment:
Given that even longterm Debian users are confused about which codename is used for which release, maybe make this:

    Is packaged in Debian 12 (aka bookworm) and later.